EsBankAccount.Tests/Startup/BankAccountClientTests.fs

```diff
@@ -13,8 +13,8 @@ let fakeAccountId () = string Guid.Empty
 let ``Should deposit and then withdraw`` () =
     async {
         let accountId = fakeAccountId ()
-        do! deposit accountId 10m
-        do! withdraw accountId 5m
-        let! transac = ReadModelClient.transactionsOf accountId
-        transac.Length =! 2
+        do! deposit accountId 10m |> Async.Ignore
```

Comment:
consider `let! _view = deposit accountId 10m` as a way to document what you are ignoring / the intent

Reply:
Interesting yes. I'll create a new review branch for this. Thanks Ruben 🙏

Reply:
There's really no need - this is really just nits and subjective. Good to see you still doing things on this from time to time though (I have other stuff like this almost at standstill which will eventually get bits of time!)

Reply:
Knowledge shared is knowledge squared :)
https://github.com/akhansari/EsBankAccount/pull/4